src/components/Editor/Tab.js

```diff
@@ -156,6 +156,12 @@ class Tab extends PureComponent<Props> {
     const isPrettyCode = isPretty(source);
     const sourceAnnotation = getSourceAnnotation(source, sourceMetaData);
 
+    const sourceTabsWidth = document.getElementsByClassName("source-tabs");
```

Comment:
Might need a change of name as assigning DOM elements to a `width` variable feels difficult to track later.